kernel/packages/unity-interface/UnityInterface.ts

```diff
@@ -510,6 +511,36 @@ export class UnityInterface {
       resizeCanvas(_gameInstance.Module)
     }, 100)
   }
+
+  private SendMessageToUnity(object: string, method: string, payload: any = undefined) {
+    const originalSetThrew = this.Module["setThrew"]
+    const unityModule = this.Module
+    let isError = false
+
+    function overrideSetThrew() {
+      unityModule["setThrew"] = function() {
+        isError = true
+        return originalSetThrew.apply(this, arguments)
+      }
+    }
+
+    function restoreSetThrew() {
+      unityModule["setThrew"] = originalSetThrew
+    }
+
+    overrideSetThrew()
+    try {
+      this.gameInstance.SendMessage(object, method, payload)
+    } finally {
+      restoreSetThrew()
+    }
+
+    if (isError) {
+      const error = `Error while sending Message to Unity. Object: ${object}. Method: ${method}. Payload: ${payload}.`
+      defaultLogger.error(error)
+      ReportRendererInterfaceError(error, error)
```

Comment:
@menduz are all console errors reported automatically to rollbar? I'm not sure if calling this will result in the error being reported twice :S

Reply:
Based on what I read, rollbar only handles uncaught exceptions and literal calls to Rollbar.error